.eslintrc.json

```diff
@@ -4,20 +4,30 @@
     "es6": true,
     "node": true
   },
-  "extends": ["eslint:recommended", "plugin:react/recommended"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended"
+  ],
   "globals": {
     "Atomics": "readonly",
-    "SharedArrayBuffer": "readonly"
+    "SharedArrayBuffer": "readonly",
+    "test": "readonly",
```

Comment:
i believe adding `jest` to the `env` block would replace the need for `test` and `expect`. I misspoke